api2-war/src/main/java/eu/europeana/api2/v2/web/controller/ObjectController.java

```diff
@@ -407,6 +452,22 @@ private ModelAndView generateRdf(FullBean bean) {
         return new ModelAndView("rdf", model);
     }
 
+    private ModelAndView generateTurtle(FullBean bean, HttpServletResponse response) {
+        OutputStream outputStream = new ByteArrayOutputStream();
```

Comment:
Outputstream is never closed!

Reply:
Made it auto closeable

Reply:
Nice! I did notice something else just now. When there is an error we return an empty 500 response. We should return error information, so take a look at how we do that in generateJsonLd for example with `return JsonUtils.toJson(new ApiError(data.wskey, e.getClass().getSimpleName() + ": " + e.getMessage()), data.callback)`